dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04210CreateDefaultLanguageVariable.java

```diff
@@ -0,0 +1,183 @@
+            dc.addParam(field.get("isListed"));
+            dc.addParam(field.get("velocityVarName"));
+            dc.addParam(field.get("sortOrder"));
+            dc.addParam(field.get("values"));
+            dc.addParam(field.get("regexCheck"));
+            dc.addParam(field.get("hint"));
+            dc.addParam(field.get("defaultValue"));
+            dc.addParam(field.get("isFixed"));
+            dc.addParam(field.get("isReadOnly"));
+            dc.addParam(field.get("isSearchable"));
+            dc.addParam(field.get("isUnique"));
+            dc.addParam(new Date());
+            dc.loadResult();
+            i++;
+        }
+    }
+
+    /**
+     * Creates each field that is required by the Language Variable Content Type and returns a list
+     * with them.
+     * 
+     * @return The List of Language Variable fields.
+     */
+    @SuppressWarnings("unchecked")
+    private List<Map<String, Object>> getLanguageVariableFields() {
+        final boolean isRequired = Boolean.TRUE;
+        final boolean isListed = Boolean.TRUE;
+        final boolean isIndexed = Boolean.TRUE;
+        final boolean isFixed = Boolean.TRUE;
+        final boolean isReadOnly = Boolean.TRUE;
+        final boolean isSearchable = Boolean.TRUE;
+        final boolean isUnique = Boolean.TRUE;
+        Map<String, Object> keyField = setFieldElements(KeyValueContentType.KEY_VALUE_KEY_FIELD_NAME,
+                        LegacyFieldTypes.TEXT.implClass().getName(), StringPool.BLANK, "text1", isRequired, isListed, isIndexed,
+                        KeyValueContentType.KEY_VALUE_KEY_FIELD_VAR, 1, StringPool.BLANK, StringPool.BLANK, StringPool.BLANK,
+                        StringPool.BLANK, isFixed, !isReadOnly, isSearchable, !isUnique);
+        Map<String, Object> valuField = setFieldElements(KeyValueContentType.KEY_VALUE_VALUE_FIELD_NAME,
+                        LegacyFieldTypes.TEXT_AREA.implClass().getName(), StringPool.BLANK, "text_area1", isRequired, isListed,
+                        !isIndexed, KeyValueContentType.KEY_VALUE_VALUE_FIELD_VAR, 2, StringPool.BLANK, StringPool.BLANK,
+                        StringPool.BLANK, StringPool.BLANK, isFixed, !isReadOnly, isSearchable, !isUnique);
+        return list(keyField, valuField);
+    }
+
+    /**
+     * This method sets the value of each element that is needed when creating a Field.
+     * 
+     * @param fieldName
+     * @param fieldType
+     * @param fieldRelationType
+     * @param fieldContentlet
+     * @param required
+     * @param listed
+     * @param indexed
+     * @param velocityVarName
+     * @param sortOrder
+     * @param values
+     * @param checkRegex
+     * @param hint
+     * @param defaultValue
+     * @param fixed
+     * @param readOnly
+     * @param searchable
+     * @param unique
+     * @return The data required by a field.
+     */
+    private Map<String, Object> setFieldElements(String fieldName, String fieldType, String fieldRelationType,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 17 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)